packages/sirius-web/backend/sirius-web/src/test/java/org/eclipse/sirius/web/application/controllers/libraries/LibraryControllerIntegrationTests.java

```diff
@@ -174,12 +193,15 @@ public void givenProjectWithDependenciesWhenDependencyIsUpdatedThenEditingContex
         Optional<Library> optionalLibrary = this.librarySearchService.findByNamespaceAndNameAndVersion("papaya", "sirius-web-tests-data", "2.0.0");
         assertThat(optionalLibrary).isPresent();
 
+        var editingContextEventInput = new EditingContextEventInput(UUID.randomUUID(), PapayaIdentifiers.PAPAYA_EDITING_CONTEXT_ID.toString());
```

Comment:
Based on a discussion we had some time ago I updated this test to use a subscription to the editing context. I did the same for the new test I contributed.